modules/user_data/templates/install_graphdb.sh.tpl

```diff
@@ -222,71 +204,14 @@ EOF
 chmod +x /usr/bin/graphdb_backup
 echo "${backup_schedule} graphdb /usr/bin/graphdb_backup" > /etc/cron.d/graphdb_backup
 
-# Install GraphDB
-
 # https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html#connection-idle-timeout
 echo 'net.ipv4.tcp_keepalive_time = 120' | tee -a /etc/sysctl.conf
 echo 'fs.file-max = 262144' | tee -a /etc/sysctl.conf
 
 sysctl -p
 
-cd /tmp
-# sometimes host cannot be resolved
-until curl -O https://maven.ontotext.com/repository/owlim-releases/com/ontotext/graphdb/graphdb/${graphdb_version}/graphdb-${graphdb_version}-dist.zip; do
-  sleep 5
-done
-
-unzip -q graphdb-${graphdb_version}-dist.zip
-rm graphdb-${graphdb_version}-dist.zip
-mv graphdb-${graphdb_version} /opt/graphdb-${graphdb_version}
-ln -s /opt/graphdb-${graphdb_version} /opt/graphdb
-
-chown -R graphdb:graphdb /var/opt/graphdb/data /var/opt/graphdb/logs /etc/graphdb /opt/graphdb /opt/graphdb-${graphdb_version} /etc/graphdb-cluster-proxy /var/opt/graphdb-cluster-proxy/logs
-
-cat << EOF > /lib/systemd/system/graphdb.service
-[Unit]
-Description="GraphDB Engine"
-Wants=network-online.target
-After=network-online.target
-
-[Service]
-Restart=on-failure
-RestartSec=5s
-User=graphdb
-Group=graphdb
-Environment="GDB_JAVA_OPTS=-Xmx30g -Dgraphdb.home.conf=/etc/graphdb -Dhttp.socket.keepalive=true"
-ExecStart="/opt/graphdb/bin/graphdb"
-TimeoutSec=120
-SuccessExitStatus=143
-
-[Install]
-WantedBy=multi-user.target
-EOF
-
-cat << EOF > /lib/systemd/system/graphdb-cluster-proxy.service
-[Unit]
-Description="GraphDB Cluster Proxy"
-Wants=network-online.target
-After=network-online.target
-
-[Service]
-Restart=on-failure
-RestartSec=5s
-User=graphdb
-Group=graphdb
-Environment="GDB_JAVA_OPTS=-Dgraphdb.home.conf=/etc/graphdb-cluster-proxy -Dhttp.socket.keepalive=true"
-ExecStart="/opt/graphdb/bin/cluster-proxy"
-TimeoutSec=120
-SuccessExitStatus=143
-
-[Install]
-WantedBy=multi-user.target
-EOF
-
-systemctl daemon-reload
-
-systemctl enable graphdb.service
-systemctl start graphdb.service
-
+# graphdb restart is needed because the configuration file is changed and we want to apply the changes
+systemctl restart graphdb
```

Comment:
Better stop GraphDB at the beggining of this script. No point in running until this.